python/crun_python.c

```diff
@@ -401,8 +403,8 @@ container_update (PyObject *self, PyObject *args)
   if (process == NULL)
     {
       cleanup_free char *msg = NULL;
-      asprintf (&msg, "cannot parse process: %s", parser_err);
-      if (msg == NULL)
+      ret = asprintf (&msg, "cannot parse process: %s", parser_err);
+      if (UNLIKELY (ret < 0))
```

Comment:
same here, please indent the next line.  Also I think we should do the `free (parser_err)` also in case of error here